website/docs/r/dl_gateway_action.html.markdown

```diff
@@ -0,0 +1,193 @@
+- `export_route_filters` - (Optional, List) List of Export Route Filter configuration information.Applicable only for create_gateway_approve requests.
+  
+  Nested scheme for `export_route_filter`:
+  - `action` - (Required, String) Determines whether the  routes that match the prefix-set will be permit or deny
+  - `prefix` - (Required, String) IP prefix representing an address and mask length of the prefix-set
+  - `ge` - (Optional, Integer) The minimum matching length of the prefix-set
+  - `le` - (Optional, Integer) The maximum matching length of the prefix-set
+- `import_route_filters` - (Optional, List) List of Import Route Filter configuration information.Applicable only  for create_gateway_approve requests.
+   
+   Nested scheme for `import_route_filter`:
+   - `action` - (Required, String) Determines whether the  routes that match the prefix-set will be permit or deny
+   - `prefix` - (Required, String) IP prefix representing an address and mask length of the prefix-set
+   - `ge` - (Optional, Integer) The minimum matching length of the prefix-set
+   - `le` - (Optional, Integer) The maximum matching length of the prefix-set
+- `authentication_key` - (Optional, String) BGP MD5 authentication key.Applicable only create_gateway_approve for requests.
+- `bfd_interval` - (String) Minimum interval in milliseconds at which the local routing device transmits hello packets and then expects to receive a reply from a neighbor with which it has established a BFD session.Applicable only for create_gateway_approve requests.
+- `bfd_multiplier` - (String) The number of hello packets not received by a neighbor that causes the originating interface to be declared down.Applicable only for create_gateway_approve requests.
+- `connection_mode` - (Optional, String) Type of network connection that you want to bind to your direct link. Allowed values are `direct` and `transit`.Applicable only for create_gateway_approve requests.
+- `global`- (Required,Bool) Required-Gateway with global routing as **true** can connect networks outside your associated region.Applicable only for create_gateway_approve requests.
+- `metered`- (Required, Bool) Metered billing option. If set **true** gateway usage is billed per GB. Otherwise, flat rate is charged for the gateway.Applicable only for create_gateway_approve requests.
+- `resource_group` - (Optional, Forces new resource, String) The resource group. If unspecified, the account's default resource group is used.Applicable only for create_gateway_approve requests.
+- `default_export_route_filter` - (String) The default directional route filter action  that applies to routes that do not match any directional route filters.Applicable only for create_gateway_approve requests.
+- `default_import_route_filter` - (String) The default directional route filter action  that applies to routes that do not match any directional route filters.Applicable only for create_gateway_approve requests.
```

Comment:
is this a required or optional attribute?

Reply:
Fixed